test/extended/prometheus/prometheus.go

```diff
@@ -230,6 +230,44 @@ var _ = g.Describe("[sig-instrumentation] Prometheus", func() {
 				}
 				return true, nil
 			})).NotTo(o.HaveOccurred(), "possibly some services didn't register ServiceMonitors to allow metrics collection")
+
+			g.By("verifying all targets are exposing metrics over secure channel")
```

Comment:
Right above there is a line that says:
> 	// TODO: should probably be https

Should that now be https with this PR, or do we just remove that at least for CVO, as it checks the targets page as well
> verifying all expected jobs have a working target

Reply:
They should be https and some of them already are, but not all (crio, cvo) expose metrics over https as of now, so that "TODO" is still valid.

Reply:
Also that test should stay as it checks for some crucial targets to be up.